src/locales/en.json

```diff
@@ -22,7 +22,10 @@
       "title": "Uninstall this application",
       "description": "This application is bound to be removed from your Cozy %{cozyName}. Are you sure to __remove this application definitively__?",
       "cancel": "Cancel",
-      "uninstall": "Uninstall"
+      "uninstall": "Uninstall",
+      "alerts": {
```

Comment:
Same here, having `alerts` here is too much behavior related. What if towmorrow we do not use alert anymore ?
I think we should only have something like:
```js
"app_modal": {
  "uninstall": {
    "success": {
      "message": "The application has correctly been uninstalled."
     }
   }
}
```
By the way, if I understand well key structure, we have somewhere a key `app_modal.uninstall.uninstall` ?

Reply:
Yep, I didn't notice that but it's the "Uninstall" button in the modal.